ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/GlobalVariable.java

```diff
@@ -34,17 +34,30 @@
     private final QuotedIdentifier variableName;
     private final boolean isAssignableToAny;
     private final String qualifiersAndMetadata;
+    private final boolean isAssignedWithVar;
+    private final boolean isDefinedObject;
```

Comment:
As mentioned below, shouldn't be special casing anything.

Reply:
Updated in d3e7d2d557259712428f4f178b8d6bfd67325435

Reply:
Removed in d3e7d2d557259712428f4f178b8d6bfd67325435